manager-bundle/src/HttpKernel/ContaoCache.php

```diff
@@ -82,21 +77,4 @@ protected function createStore()
             'cache_tags_header' => TagHeaderFormatter::DEFAULT_HEADER_NAME,
         ]);
     }
-
-    /**
-     * Checks if the Request includes authorization or other sensitive information
-     * that should cause the cache to be bypassed.
-     */
-    private function isRequestPrivate(Request $request): bool
-    {
-        if ($request->headers->has('Authorization')) {
```

Comment:
Are we no longer handling `Authorization` headers? Maybe we should check for them as well?

Reply:
Yeah I wasn't sure. If so, it should go to the `CacheListener` (or whatever it's going to be named). What do you think @ausi @leofeyer ?

Reply:
ping @leofeyer @ausi

Reply:
An `Authorization` header sure makes the request uncacheable, doesn't it? So we should handle it IMHO.

Reply:
No we should not handle it. There is no case in Contao where you can restrict content to an `Authorization` header being present. The only case I can imagine is some `.htaccess` protection during development and that's exactly where it is irrelevant for us. So if someone really wants to build a module/fragment that needs this header, they must also `Vary` on it.